static/src/javascripts/projects/commercial/modules/liveblog-adverts.ts

```diff
@@ -167,11 +168,23 @@ export const init = (): Promise<void> => {
 
 	return fastdom
 		.measure(() => {
+			const isServerSideAdsMode =
```

Comment:
Could we possibly use the server-side test itself to check which mode we're in?

```typescript
window.guardian.config.tests.serverSideLiveblogInlineAds === 'variant'
```

Reply:
Yes, much better! I didn't realise we could access server side AB tests on the config object

Reply:
I think you can just set the property directly in the test. There's some examples of doing this in the existing tests [e.g. here for a switch](https://github.com/guardian/frontend/blob/4dd4a9095a80d8c2783bc18ab7c2042b85f15b54/static/src/javascripts/projects/commercial/am-i-used.spec.ts#L23).